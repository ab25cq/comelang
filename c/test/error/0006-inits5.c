
/*
PATTERN: incomplete
PATTERN: c:6:
*/
struct S s = {1};

