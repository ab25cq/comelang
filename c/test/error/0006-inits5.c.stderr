cannot initialize an incomplete struct/union at test/error/0006-inits5.c:6:14
struct S s = {1};
             ^
